Automattic/jetpack
# Added a new callable for loaded PHP modules.

This PR introduces a new callable for Jetpack Sync that would add loaded PHP extensions on the user's site.

## Proposed changes:
* Added a callable for `get_loaded_extensions()`.

### Other information:
- [*] Have you written new tests for your changes, if applicable?

## Jetpack product discussion
* N/A

## Does this pull request change what data or activity we track or use?
* Adds a new callable that receives minor details of the PHP configuration.

## Testing instructions:





* Sandbox your Jetpack site.
* Add this code to currently running code on the sandbox to whitelist the new callable
* Use the cache site explorer to see the options.
* Locate the `jetpack_callable_get_loaded_extensions`.
